cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -2219,32 +2227,44 @@ describe('CourseOutlinePage', function() {
                 expectShowCorrectness('never');
             });
 
-            it('shows validation error on due number of weeks', function() {
+            it ('does not show relative date input when assignment is not graded', function() {
```

Comment:
In this test, it would be great if you can show the change in behavior. Like having 
```
setEditModalValuesForCustomPacing('Lab', '');
expect($('#relative_date_input').css('display')).not.toBe('none');
expect($('#grading_type').val()).toBe('notgraded');
setEditModalValuesForCustomPacing('notgraded', '');
expect($('#relative_date_input').css('display')).toBe('none');
expect($('#grading_type').val()).toBe('notgraded');
```
So we can see how it shows up when graded and then doesn't show up when we switch to not graded